lib/App/Critique/Command/process.pm

```diff
@@ -85,7 +85,22 @@ MAIN:
         info('Running Perl::Critic against (%s)', $path);
         info(HR_LIGHT);
 
-        my @violations = $self->discover_violations( $session, $file, $opt );
+        # perlcritic can fail, so lets guard against it and let the user
+        # decide if they want to carry on
+        my @violations = eval {
+            $self->discover_violations( $session, $file, $opt )
+        };
+        if ($@) {
+            info(HR_ERROR);
+            warn($@);
+            info(HR_LIGHT);
+            my $should_review = prompt_yn(
+                BOLD(sprintf 'A error has occurred do you want to continue?', (scalar @violations)),
+                { default => 'y' }
```

Comment:
You can drop the `(scalar @violations)` there, it is not being used in the `sprintf` string, so it is just superfluous.